### DISCUSSION TITLE
What parameters can be used in the DASHBOARD_UIALERTS

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#37823] [Q&A]
Ive been lookong into this part of DASHBOARD_UIALERTS  and was wondering what kind of categories or other parameters i can use in this part.
I now for example have:

from airflow.www.utils import UIAlert

```
DASHBOARD_UIALERTS = [
        UIAlert("No error messages today.", category="error"),
        UIAlert("No warning messages today.", category="warning"),
        UIAlert("No info messages today.", category="information"),
]
```

This gives me a red, yellow and white banner.
But i see also blue or green if i take actions in Airflow itself.

Is there documentation on this?

**Comment by potiuk** [comment]
Not documented in airflow other than the categories you see - they come from FlaskAppBuilder and you should look there, likely requires you to dive deep in the sources of FAB. If you find out, it would be great if you can contribute such documentation back - this the way how such documentation gets added -when somene interested finds  a need for it, digs deeper, finds out and contributes it back as a PR (most of our > 2800 contributors are this kind of contributions).
